safehttp/cookie.go

```diff
@@ -38,7 +38,7 @@ func NewCookie(name, value string) *Cookie {
 		&http.Cookie{
 			Name:     name,
 			Value:    value,
-			Secure:   true,
+			Secure:   !isLocalDev,
```

Comment:
Please document this behavior.

Reply:
I'll do this in the HSTS docs and NewCookie funcs